src/components/DisplayAccounts.jsx

```diff
@@ -35,70 +36,89 @@ export default function DisplayAccounts({ accounts, ...props }) {
   const { t } = useTranslation()
   const settings = useSettings()
 
+  const parseTotalBalanceFromRawStringOrEmpty = (rawBalanceString, source) => {
+    try {
+      return parseTotalBalanceString(rawBalanceString).totalBalance
+    } catch (e) {
+      console.warn(`cannot determine balance format for ${source}`)
+      return ''
```

Comment:
Returning `''` would result in no balance being shown for this account/branch/address right?

Reply:
Yes it would look like this:
<img src="https://user-images.githubusercontent.com/3358649/168823956-5a24094c-492f-41b7-9116-ce6d6080a899.png" width=300 />

Instead of this:
<img src="https://user-images.githubusercontent.com/3358649/168824067-bd6cc947-b68b-4b3a-baee-218f6723fd3f.png" width=300 />

The only case this can occur, if I am not mistaken, is, if the format has changed again. Which hopefully does not happen, but better safe than sorry. FYI, `amount` value in the `branches[]->entries[]` object does not change and is still a btc value represented as string, e.g. `{ amount: "150.00000000", [...] }`. Hope that also does not change (otherwise the UI will break again as this is not handled).